pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -208,10 +179,31 @@ func (a *Actuator) Create(ctx context.Context, cluster *clusterv1.Cluster, machi
 	if err := a.reconcileLBAttachment(scope, machine, i); err != nil {
 		return errors.Errorf("failed to reconcile LB attachment: %+v", err)
 	}
-	a.log.Info("Create completed", "machine-name", machine.Name)
+
+	log.Info("Create completed")
+
 	return nil
 }
 
+func (a *Actuator) isNodeJoin(log logr.Logger, cluster *clusterv1.Cluster, machine *clusterv1.Machine) (bool, error) {
```

Comment:
can we have test coverage for this?

Reply:
Yes, I can add it. This was extracted from within the Create method, which made it hard to test.